modules/programs/firefox.nix

```diff
@@ -169,7 +169,10 @@ let
     else if isDarwin then
       package
     else if versionAtLeast config.home.stateVersion "19.09" then
-      package.override (old: { cfg = old.cfg or { } // fcfg; })
+      package.override (old: {
+        cfg = old.cfg or { } // fcfg;
+        extraPolicies = cfg.policies;
```

Comment:
This is a breaking change if users use `programs.firefox.package = pkgs.firefox.override { extraPolicies = { ... } }` to specify their policies before.  A release note should be added.

Reply:
@rycee this change is part of the already released 23.11. Should old release notes be modified?

Reply:
Changing the implementation to `extraPolicies = old.extraPolicies // cfg.policies;` makes this almost not breaking.

Reply:
Adding to the existing extra policies seems like a good idea. Perhaps need to add a guard like in the `cfg` line above, though? I.e.

``` nix
extraPolicies = old.extraPolicies or { } // cfg.policies;
```

Does that seem ok? With that change I guess it wouldn't be necessary to add to the changelog? Regardless, I have nothing against adding to the 23.11 changelog, if it helps clarify…